server/application/rest/service_shifts.py

```diff
@@ -10,42 +12,65 @@
 from application.rest.work_shift import HTTP_STATUS_CODES_MAPPING
 from responses import ResponseTypes
 from serializers.service_shift import ServiceShiftJsonEncoder
-import json
 
-service_shift_bp = Blueprint('service_shift', __name__)
+service_shift_bp = Blueprint("service_shift", __name__)
+
 
-@service_shift_bp.route('/service_shift', methods=['GET', 'POST'])
+@service_shift_bp.route("/service_shift", methods=["GET", "POST"])
 @cross_origin()
-def service_shift():
+def handle_service_shift():
     """
-    Handles POST to add service shift, GET to list all shifts for a shelter.
+    Handles POST to add service shifts and GET to list all shifts for a shelter.
     """
     repo = ServiceShiftsMongoRepo()
-    if request.method == 'GET':
-        shelter_id_string = request.args.get('shelter_id')
-        shelter_id = int(shelter_id_string) if shelter_id_string else None
-        shifts_as_dict = service_shifts_list_use_case(repo, shelter_id)
-        shifts_as_json = [
-            json.dumps(service_shift, cls = ServiceShiftJsonEncoder)
-            for service_shift in shifts_as_dict
-        ]
+
+    if request.method == "GET":
+        shelter_id_str = request.args.get("shelter_id")
+        filter_start_after_str = request.args.get("filter_start_after")
+
+        # Ensure proper conversion to int, handling empty or invalid cases
+        shelter_id = int(shelter_id_str) if shelter_id_str and shelter_id_str.isdigit() else None
```

Comment:
```suggestion
        shelter_id = (
            int(shelter_id_str)
            if shelter_id_str and shelter_id_str.isdigit()
            else None
        )
```
This should resolve the first linting error